src/http/server.clj

```diff
@@ -293,6 +294,24 @@
          (log/infof "\"%s %s HTTP/1.1\" %s %s" method'  uri status len))
        response))))
 
+(defn pw-prompt []
+  (str (do (println "Password: ") (.readPassword (System/console)))))
```

Comment:
We also need to discard empty password here.

Reply:
If they enter no password, the empty string is just passed along and the server presumably rejects it.  How do you want to proceed if they enter no password?  Prompt them again?  I'm not sure what discard means in this sense.